lib/htsget-server/docker-compose.yml

```diff
@@ -29,5 +31,14 @@ services:
         - "traefik.docker.lbswarm=true"
         - "traefik.http.routers.htsget-app.rule=Host(`htsget-app.${CANDIG_DOMAIN}`)"
         - "traefik.http.services.htsget-app.loadbalancer.server.port=${HTSGET_APP_PORT}"
+    secrets:
+      - source: minio-access-key
+        target: access
+      - source: minio-secret-key
+        target: secret
+      - source: selfsigned-site-pem
+        target: cert.pem
     logging: *default-logging
+    environment:
+      HTSGET_TEST_KEY: "hoodlebug"
```

Comment:
What is TEST_KEY for?

Reply:
sorry, this is clearer if you're on the correct version of htsget. For htsget's pytest, you need to have a test key to bypass OPA, and it's set as an env var that both the htsget server and its pytest have access to.

Reply:
(I updated the commit for htsget below)